src/sp/it/pl/util/action/Action.java

```diff
@@ -533,7 +536,7 @@ private static void stopGlobalListening() {
 	}
 
 	/**
-	 * Returns true iff global shortcuts are supported at running platform.
+	 * Returns true if global shortcuts are supported at running platform.
```

Comment:
Not a typographical error. Reverting back. "Iff" means "if and only if" which is the correct phrase in such case (and most of the time when |if" is used in a javadoc return section).

Reply:
oh, you never stop learning ^^